Grbl_Esp32/src/Spindles/Laser.cpp

```diff
@@ -31,8 +31,8 @@ namespace Spindles {
     void Laser::config_message() {
         grbl_msg_sendf(CLIENT_SERIAL,
                        MsgLevel::Info,
-                       "Laser spindle on Pin:%s, Freq:%dHz, Res:%dbits Laser mode:%s",
-                       pinName(_output_pin).c_str(),
+                       "Laser spindle on Pin:%s, Freq:%.2fHz, Res:%dbits Laser mode:%s",
```

Comment:
"Laser spindle on Pin:%s, Freq:%.dHz, Res:%dbits Laser mode:%s",

_pwm_freq is a unit32t